app/api/events.py

```diff
@@ -575,3 +580,10 @@ def clear_export_urls(event):
     event.xcal_url = None
     event.pentabarf_url = None
     save_to_db(event)
+
+
+@event_location_routes.route('/most-used-locations', methods=['GET'])
+def most_used_location():
+    locations = db.session.query(Event.location_name).group_by(Event.location_name). \
```

Comment:
For line continuation, you should try avoiding back slashes, instead break on parenthesis. Something like this
```suggestion
    locations = db.session.query(Event.location_name).group_by(
    Event.location_name).
```

Reply:
Yeah, don't need backslashes when parenthesis are present.